### PR TITLE
Docs 8840 docs 8842 9623 dashboard fixes

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/app-dashboards.adoc [hunk 1]
```diff
@@ -50,13 +50,13 @@ Built-in dashboards include various pages to view different types of charts:
...
 
 * Inbound - Total Requests by Response Type
-* Average Response Time
-* Response Time by Endpoint
-* Total Requests by Endpoint
+* Inbound - Average Response Time
+* Inbound - Response Time by Endpoint
+* Inbound - Total Requests by Endpoint
 * Inbound - Response Time graphs (99, 90, 75, and 50 percentiles)
-* Maximum Response Time
```
Comment (myuin-ms) [inline modules/ROOT/pages/app-dashboards.adoc:57]: Take this line out. It's repeated below.

**Review by myuin-ms — COMMENTED** [review]
In the "Outbound" tab, "ports" chart is listed twice. I realize it's an existing bug, let's fix it together now that I see it.

**Comment by myuin-ms** [comment]
In addition, the performance section is still incorrect. There are still several inconsistencies between the doc and the actual product. If you need an account with data to see the actual charts, please ping the team.

**Comment by myuin-ms** [comment]
Infrastructure charts are still off. What's documented does not match the product.

File: modules/ROOT/pages/app-dashboards.adoc [hunk 2]
```diff
@@ -50,13 +50,13 @@ Built-in dashboards include various pages to view different types of charts:
...
+* Inbound - Maximum Response Time
```
Comment (sarathecoubian) [inline modules/ROOT/pages/app-dashboards.adoc:57]:
```suggestion
```

**Comment by sarathecoubian** [comment]
> Infrastructure charts are still off. What's documented does not match the product.

Updated. RE: Performance, the Outbound - Response Time Grouped by HTTP Endpoint should not have "grouped" in it. We removed that elsewhere.

File: modules/ROOT/pages/app-dashboards.adoc [hunk 3]
```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
...
 Outbound - Slow Requests by Flow::
 The sum of outbound requests whose response time is greater than 1000 ms, grouped by flow. Cell color varies according to the following thresholds:
-
 * Green if the response time is less than 300 ms
 * Yellow if the response time is greater than 300 ms and less than 600 ms
 * Red if the response time is greater than or equal to 600 ms
 
+Inbound - Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
Comment (dhirsch1) [inline modules/ROOT/pages/app-dashboards.adoc:340]: I don't think there should be a period here and elsewhere where there is similar text.

**Review by dhirsch1 — DISMISSED** [review]

File: modules/ROOT/pages/app-dashboards.adoc [hunk 4]
```diff
@@ -446,18 +480,18 @@ The average number of requests failures of connectors of type `other`.
...
+
 Inbound - Average Response Time::
 The average response time of inbound requests.
 
-Inbound - Average Response Time Grouped by Endpoint::
+Inbound - Average Response Time by Endpoint::
 The average, maximum, and minimum response times, grouped by top occurring endpoint. For more information about which endpoints appear, see the section about xref:monitoring::app-dashboards.adoc#endpoint-operation-metrics[endpoint and operation data collection]. Cell color varies according to the following thresholds:
 * Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
 * Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms.
 * Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/app-dashboards.adoc:493]:
```suggestion
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms
* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

File: modules/ROOT/pages/app-dashboards.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/app-dashboards.adoc:340]:
```suggestion
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

File: modules/ROOT/pages/app-dashboards.adoc [hunk 5]
```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
...
-
 * Green if the response time is less than 300 ms
 * Yellow if the response time is greater than 300 ms and less than 600 ms
 * Red if the response time is greater than or equal to 600 ms
 
+Inbound - Response Time by Flow::
+The average, minimum, and maximum inbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
+* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms.
+* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/app-dashboards.adoc:342]:
```suggestion
* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms
* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

File: modules/ROOT/pages/app-dashboards.adoc [hunk 6]
```diff
@@ -324,20 +331,47 @@ The sum of inbound requests whose response time is greater than 1000 ms, grouped
...
+
+Outbound - Response Time by Flow::
+The average, minimum, and maximum outbound response time, grouped by flow.
+* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
+* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms.
+* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms.
```
Comment (sarathecoubian) [inline modules/ROOT/pages/app-dashboards.adoc:348]:
```suggestion
The average, minimum, and maximum outbound response time, grouped by flow.
* Average: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
* Maximum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 500 ms; red if the response time is greater than or equal to 500 ms
* Minimum: Green if the response time is less than 300 ms; yellow if the response time is greater than 300 ms and less than 600 ms; red if the response time is greater than or equal to 600 ms
```

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**